conf/distro/nilrt.conf

```diff
@@ -4,7 +4,7 @@ DISTRO_VERSION = "10.0"
 
 DISTRO_CODENAME = "${LAYERSERIES_COMPAT_meta-nilrt}"
 
-NILRT_FEED_NAME ?= "next"
+NILRT_FEED_NAME ?= "2023Q2"
```

Comment:
Shouldn't this be "next/2022Q2" so that devs building locally can access the feeds

Reply:
When I pinged @amstewart he said it should be like hardknott. Maybe I misunderstood; I had thought that the changes in AzDO to add the `next/` prefix would be enough since nilrt.inc here is using download.ni.com instead of nickdanger.

Reply:
`ni-org.conf` (included when `--org` arg is used for `ni-oe-init-build-env`) sets feeds to nickdanger.

Reply:
Should the `next/` prefix should be added in that file, then?

Reply:
The server domain set by the org.conf files isn't relevant here. None of that logic is changing for the kirkstone mainline.

This variable only sets the default feed namespace in the OE-built images. So it is largely inconsequential for local builders, because they will be building the feed contents locally and would have to write their own opkg feed configs from scratch, if they want to do a local package installation.

The build servers [set this value](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=/src/rtos/includes/bitbakeRules.mak&version=GBmain&line=105&lineEnd=106&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) to what is coordinated through the AZDO `REPO_EXTERNAL_VERSION_NOSPACE` variable. So it is also overwritten there.

The more I think about this variable, the more I think we might just want to deprecate it. @mike-petersen-ni Maybe we just drop this PR for now and let it stay on `next`.